docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md

```diff
@@ -0,0 +1,229 @@
+# Configure OE SDK SGX on Linux in non-ACC Machines
+
+## Disclaimer
+This document is to provide a viable solution to enable Open Enclave SGX DCAP remote attestation to run on non-Azure Confidential Computing (ACC) machines. It relies on several Intel components and services which are subject to Intel's changes.
+
+## 1. Platform requirements
+- Ubuntu 18.04-LTS 64-bits (should also work on 16.04 LTS but hasn't been tested).
+- SGX1 capable system with Flexible Launch Control support. This feature is only available on Intel Coffee Lake processor (8th gen) or newer.
+- Strongly recommend to update your BIOS to newest version before start. With the setup described by this document, all attestation will be against the most recent collateral. Old BIOS versions, which may have lower CPU SVN, will cause attestation to fail.
+
+## 2. Set up openenclave environment
+### 2.1 Clone Open Enclave SDK repo from GitHub
+Use the following command to download the source code.
+
+```bash
+git clone https://github.com/openenclave/openenclave.git
+```
+
+This creates a source tree under the directory called openenclave.
+
+### 2.2 Install project requirements
+
+First, change directory into the openenclave repository:
+```bash
+cd openenclave
+```
+
+Ansible is required to install the project requirements. If not already installed, you can install it by running:
+```bash
+sudo scripts/ansible/install-ansible.sh
+```
+
+Then run the following command to install the dependency:
+```bash
+ansible-playbook scripts/ansible/oe-contributors-setup.yml
+```
+NOTE: The Ansible playbook commands from above will try to execute tasks with sudo rights. Make sure that the user running the playbooks has sudo rights, and if it uses a sudo password add the following extra parameter --ask-become-pass.
+
+## 3. Set up Intel DCAP Quote Provider Library (QPL):
+### 3.1 Install Intel DCAP Quote Provider by using its Debian package
+The [libsgx-dcap-default-qpl directory](https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/ubuntuServer18.04/debian_pkgs/libs/libsgx-dcap-default-qpl/) lists all different version of libsgx-dcap-default-qpl, please download the most recent version that matches your OS version. For Ubuntu 18.04 (code name [Bionic Beaver](https://wiki.ubuntu.com/BionicBeaver)), please download the version libsgx-dcap-default-qpl_{version}-bionic1_amd64.deb. For Ubuntu 16.04 (code name [Xenial Xerus](https://wiki.ubuntu.com/XenialXerus)), please download the version libsgx-dcap-default-qpl_{version}-xenial1_amd64.deb.
```

Comment:
If you add Intel's apt repo you can install the package from there instead of downloading the deb. Should we recommend that instead?

Reply:
Thanks for the suggestions. I have added the content on installing qpl by directly using APT as a recommended approach.